Parsers/Motivation Quote.js

```diff
@@ -0,0 +1,20 @@
+/*
+activation_example: !quote
+regex: !quote
+flags: gmi
+order: 100
+stop_processing: true
+*/
+
+const quotes = [
```

Comment:
These types of parsers either need to have so many quotes already fully loaded into them or they should access an api to do so.